doc/docs-revamp/dev-onboarding-quick-setup.md

```diff
@@ -0,0 +1,375 @@
+
+## Setting up and testing your development environment
+
+### Prerequisites
+
+#### Hardware and OS requirements
+
+**Processing power**
+- Minimum: Intel Core i5 or AMD equivalent
+- Recommended: Intel Core i7 or AMD Ryzen 7 
+
+**RAM**
+- Minimum: 8GB RAM
+- Recommended: 16GB RAM or more
+
+**Operating system**
+- Linux: A recent distribution of Linux such as Ubuntu 20.04 LTS or later is preferred due to its widespread use in the Haskell community and its compatibility with Plutus tooling.
+- macOS: macOS Catalina (10.15) or later
+- Windows: While Windows can be used for Haskell development, it is less common and might require additional setup such as using Windows Subsystem for Linux (WSL2) to provide a Linux-compatible environment.
+
+### Tools
+
+To start developing with Haskell and Plutus, you'll need to set up your development environment with the essential tools. This section will guide you through installing and configuring the necessary components. 
+
+#### Nix
+
+Nix is a package manager that provides reproducible and reliable software environments. It is particularly useful in Plutus development for managing dependencies and ensuring consistent development environments across different machines. 
+
+While not strictly required, Nix is highly recommended because of its effective management of complex library dependencies. 
+
+To install Nix, follow the instructions on the [official Nix website](https://nixos.org/download.html). 
+After installing, configure Nix to use IOHK’s binary caches to speed up the build process. For important details and context, see these two documents: 
+- [Contributing to Plutus > Installing and setting up Nix](https://github.com/IntersectMBO/plutus/blob/master/CONTRIBUTING.adoc),
+- [Nix setup guide](https://github.com/input-output-hk/iogx/blob/main/doc/nix-setup-guide.md): instructions for installing and configuring Nix to work with projects at IOG.
+
+#### Haskell tool stack ("Stack")
+
+Stack is a cross-platform program for developing Haskell projects. It handles dependency management, building, and testing your Haskell code. 
+
+To install the Stack: 
+1. Visit the [official Haskell Stack website](https://docs.haskellstack.org/). 
+2. Follow the installation instructions for your operating system (Linux, macOS, or Windows).
+3. Verify your installation by running `stack --version` in your terminal.
+
+#### GHC (Glasgow Haskell Compiler)
+
+- GHC is the main compiler for Haskell with many extensions and optimizations. 
+- To install GHC, run `stack setup`. Stack will automatically install the appropriate version of GHC for your project. 
+
+#### Cabal
+
+Cabal is a system for building and packaging Haskell libraries and programs. It works well with Stack and is used to manage project dependencies and build configurations. Cabal is installed automatically when you install Stack. 
+
+To verify your Cabal installation, run `cabal –version`. 
+
+### Git
```

Comment:
We definitely don't need to teach people how to install Git. It is so basic and not specific to Plutus.

Reply:
Makes sense.